assets/runtime/functions

```diff
@@ -419,7 +419,11 @@ gitlab_configure_mailroom() {
     IMAP_SSL \
     IMAP_STARTTLS \
     IMAP_MAILBOX \
-    IMAP_TIMEOUT
+    IMAP_TIMEOUT \
+    IMAP_METHOD \
+    IMAP_TENANT_ID \
+    IMAP_CLIENT_ID \
+    IMAP_CLIENT_SECRET
```

Comment:
Please add these parameters in the prescribed format between "-e"/{{IMAP_TIMEOUT}}/d"\" and "${GITLAB_CONFIG}" at the end of the [arguments of `exec_as_git sed -i` above (line 400)](https://github.com/sameersbn/docker-gitlab/pull/2610/files#diff-934fe23ac2af75a265cbd6e8da77882402932d36cfe726f9b11f1a43d0bdc5bfR400-R409). This is done to remove unnecessary settings in gitlab.yml if IMAP is disabled.

Reply:
Instead of "delete if IMAP is disabled", perhaps a better process would be "delete if the corresponding parameter is not set". I will give it some more thought...

Reply:
Done as suggested